shared/js/background/classes/site.es6.js

```diff
@@ -69,7 +70,7 @@ class Site {
      * and set the new site whitelist statuses
      */
     setWhitelistStatusFromGlobal () {
-        let globalwhitelists = ['whitelisted']
+        let globalwhitelists = ['whitelisted', 'whitelistOptIn']
```

Comment:
We're creating a new list that mirrors the whitelist for white list opt ins. This will save which sites users have added to the whitelist and opted into sharing breakage data on. We are doing this so that we can attach the domain to the ept.on pixel when a user unwhitelists a site, which we think is a pretty good sign that the site isn't broken.